docker.Makefile

```diff
@@ -119,7 +119,9 @@ vendor: build_dev_image
 	docker rm -f docker-app-vendoring
 	$(warning You may need to reset permissions on vendor/*)
 
-check-vendor: build_dev_image
+clean-vendor-cache:
+	docker rm -f docker-app-vendoring || echo ""
+	docker volume rm -f docker-app-vendor-cachecheck-vendor: build_dev_image
```

Comment:
You are missing a `\n` here so the `check-vendor` rule is on the end of this line...